vividus-tests/src/main/resources/story/integration/SearchAttributes.story

```diff
@@ -9,5 +9,5 @@ Then number of elements found by `checkboxName(One)` is equal to `1`
 
 Scenario: Verify tagName(...) search attribute and index(...) filter
 Given I am on a page with the URL '${vividus-test-site-url}/elementState.html'
-When I save `id` attribute value of element located `tagName(button):i->filter.index(2)` to SCENARIO variable `elementId`
+When I save `id` attribute value of element located `tagName(button)->filter.index(2)` to SCENARIO variable `elementId`
```

Comment:
why does it require changes?

Reply:
there are no invisible buttons on the elementState page, it worked because visibility was forced to all